html/canvas/element/manual/imagebitmap/createImageBitmap-resolves-in-task.any.js

```diff
@@ -0,0 +1,117 @@
+// META: title=createImageBitmap resolves in a task
+// META: script=/common/media.js
+// META: script=./common.sub.js
+function makeWorkerImageBitmap() {
+    return makeOffscreenCanvas().then(canvas => {
+        return createImageBitmap(canvas);
+    });
+}
+var imageSourceTypes = self.GLOBAL.isWorker() ?
+    [
+        { name: 'an OffscreenCanvas', factory: makeOffscreenCanvas },
+        { name: 'an ImageData', factory: makeImageData },
+        { name: 'an ImageBitmap', factory: makeWorkerImageBitmap },
+        { name: 'a Blob', factory: makeBlob('/images/pattern.png') },
+    ] :
+    imageSourceTypes;
+
+var testFuncs = {
+    reject_sync: (promiseFunc, source, t) => {
+        return new Promise((resolve, reject) => {
+            let rejected = false;
+            promiseFunc(source).then(function() {
+                reject(new Error('Expected this call to reject'));
+            }, (err) => {
+                rejected = true;
+            });
+            Promise.resolve().then(() => {
+                try {
+                    assert_equals(rejected, true)
+                    resolve(t);
+                } catch (err) {
+                    reject(err);
+                }
+            })
+        });
+
+    },
+    resolve_async: (promiseFunc, source, t) => {
+        return new Promise((resolve, reject) => {
+            let taskRan = false;
+            promiseFunc(source).then(function() {
+                try {
+                    assert_equals(taskRan, true)
+                    resolve(t);
+                } catch (err) {
+                    reject(err)
+                }
+            }, t.unreached_func('Expected this call to resolve'))
+            Promise.resolve().then(() => {
+                taskRan = true;
+            });
+        });
+    },
+    reject_async: (promiseFunc, source, t) => {
+        return new Promise((resolve, reject) => {
+            let taskRan = false;
+            Promise.resolve().then(function() {
+                promiseFunc(source).then(
+                    t.unreached_func('Expected this call to reject'),
+                    function() {
+                        try {
+                            assert_equals(taskRan, true)
```

Comment:
Some assertion messages here would make the results of https://wpt.fyi/results/html/canvas/element/manual/imagebitmap/createImageBitmap-resolves-in-task.any.html?diff&filter=ADC&run_id=6296029702127616&run_id=5153843161858048 easier to read.